docs/Reference/Pantheon-API-Methods.md

```diff
@@ -1307,9 +1626,6 @@ Returns transaction information for the specified block number and transaction i
 
 Object - [Transaction object](Pantheon-API-Objects.md#transaction-object), or `null` when no transaction is found.
 
-!!!note
```

Comment:
Why removing this node ? It seems important to me.

Reply:
It felt like assumed knowledge for the intended audience but maybe not?  Can discuss on our sync tonight.